librocksdb_sys/src/lib.rs

```diff
@@ -976,6 +980,46 @@ extern "C" {
                                                       props: *mut DBTablePropertiesCollection,
                                                       errptr: *mut *mut c_char);
 
+    // Get All Key Versions
+    pub fn crocksdb_keyversions_create() -> *mut DBKeyVersions;
+
+    pub fn crocksdb_keyversions_destroy(kvs: *mut DBKeyVersions);
+
+    pub fn crocksdb_keyversions_iterator_create(kv: *mut DBKeyVersions)
```

Comment:
We know the DBKeyVersions and its count, why not using a function like crocksdb_keyversions_get(kvs: * DBKeyVersions, index: usize) to get the version directly?